### DISCUSSION TITLE
Don't tell users to `pip install vorta`

### DISCUSSION THREAD

**Discussion by ktosiek** [borgbase/vorta#831] [General]
**Is your feature request related to a problem? Please describe.**
Current documentation for installing Vorta on Linux suggests running `pip3 install vorta`, which might cause problem with user's system Python installation (if it even succeeds, but users are used to slapping `sudo` in front of failing commands :-)).

**Describe the solution you'd like**
The documentation should suggest a safer installation method. I've tested installing with [pipx](https://pipxproject.github.io/pipx/) on Ubuntu 20.04, with pipx installed from the repository, and it works well.

**Comment by Hofer-Julian** [comment]
I'd be fine with that.

@m3nu ?

**Comment by m3nu** [comment]
Users who use `pip` generally know how to maintain their Python env. E.g. on macOS you would use Homebrew Python, which is different from system Python. (not sure there is even a system Python these days)

On most project pages, I usually see just `pip install ..`. Users will know themselves whether they want to use a specific env or the `--user` option.

I'm OK to add a very short note on using a venv or `--user` option, but wouldn't link to a specific pip replacement as there are too many. There was a big controversy about `pipenv` some time ago, when they oversold themselves as "the standard way", which isn't the case.

**Comment by m3nu** [comment]
PS: The docs for Vorta are in this Github repo: https://github.com/borgbase/vorta.borgbase.com

**Comment by ktosiek** [comment]
Would it make sense to describe steps to create a venv, install vorta there, and link `$venv/bin/vorta` to `~/.local/bin/`? That's pretty much what tools like pipsi or pipx do, and should be a nice and safe installation method.

pipx is not really a pip replacement, and it's not a development tool like pipenv. It is meant for installing Python applications, that's why I've suggested it.

Developers who use pip generally know how to use it, and when/how to use venvs. Here I'm concerned about users who are not Python developers, and don't have to know how pip works - they just want to install Vorta. #565 is an example of this - the first install method caused problems, so the user tried the next one.
